app/views/partners/show.html.erb

```diff
@@ -100,11 +100,12 @@
           </div>
           <h3 class="udl udl--fw allcaps h4">Contact</h3>
           <div class="small">
-            <%= render_component "contact_details",
+            <%= render ContactDetails.new(
               email: place.public_email,
```

Comment:
I guess I haven't tackled this one, but I guess what's the desired approach here. Should it just take the partner still, or have the option to override email, phone & url as this is doing?

I guess it's not quite right too that the `is_valid_phone` checks the partner valid phone, when the place phone is being used too.

Reply:
Yeah a bit confusing - we used to have sep partner and place models but it got too messy so we merged into one. Now it's messy in a diff way lol. A place is now a partner that represents a physical location for the event. So yes - this should be partner - but this whole logic needs a bit of a review and to make explicit more.

Reply:
Given our priority is to remove mountain_view its prob best to not mess with this as much as possible!

Reply:
How's this for a fix? I've pushed the changes, basically takes the partner model, but allows some optional params?

Reply:
so the easiest way to fix the tests is to drop a `byebug` in the offending place then run the specific test - it should kick you into a console where you can poke around. im sure theres more elegant ways but it works for me!

this is looking great tho!!

Reply:
fyi when youre working on views you can also put a console in a view with `<% console %>`, or chuck some garbage in the controller to make it error and itll throw up a console too

Reply:
Seems like the tests are system tests now failing on this, I'm not sure this is code I touched, so it's interesting. I see a ticket regarding flaky tests, is this what it's regarding?

Failing test is `CreateAdminUsersTest#test_visiting_the_index:`. Attached the corresponding image.

I don't see where this is getting called from though? I see this string in the devise yml file, but I don't see it called from anywhere in the flash array.

![failures_test_visiting_the_index](https://github.com/user-attachments/assets/aa9ddaf1-67eb-4e94-98f8-582bbd3839b9)

Reply:
and welcome to hopefully the last bit of annoyance right now - at some point our system tests (i.e. ones that do the browser emulation and use js) got flaky. this is one. you prob have a good sense of the current milestone now lol. ill rerun it!

Reply:
I did re-run it 3 times, and it kept flaking, so brought it back here, but fingers crossed this time eh!

Reply:
tbh i think we should prob comment them out from running on CI. theyre SUCH a pain and are slowing down develoment a ton and dont cover _that_ much functionality. feel free to do it as part of this (change line 55 in the Makefile to just be `test` not `test:system`) and ill document in the ticket to handle this.

it started about 1 in 20 times and now it feels like its 50/50 or worse, big cry

Reply:
yep that works good shout!

Reply:
issue here for the record: #2704. really tho i think we want to address this with this: #2573